packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenuContent.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+
+export interface OverflowMenuContentProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenuContent */
+  className?: string;
+  /** Indicates breakpoint at which to switch between horizontal menu and vertical dropdown */
+  breakpoint?: 'Md' | 'Lg' | 'Xl';
+}
+
+export interface OverflowMenuContentState extends React.HTMLProps<HTMLDivElement> {
+  isHidden: boolean;
+}
+
+export class OverflowMenuContent extends React.Component<OverflowMenuContentProps, OverflowMenuContentState> {
+  constructor(props: OverflowMenuContentProps) {
+    super(props);
+    this.state = {
+      isHidden: false
+    }
+  }
+
+  componentDidMount() {
+    this.handleResize();
+    window.addEventListener('resize', this.handleResize);
+  }
+
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize);
+  }
+
+  handleResize = () => {
+    const { breakpoint } = this.props;
+    const breakpointWidth = (breakpoint === 'Md')
+      ? 768
+      : (breakpoint === 'Lg')
+        ? 992
+        : 1200;
+    this.setState({ isHidden: window.innerWidth < breakpointWidth });
```

Comment:
this will set state on every resize. I am not super sure, but i think this would cause unneeded re-renders. Perhaps a conditional might make sense, e.g
```
const isHidden = window.innerWidth < breakpointWidth;
this.state.isHidden !== isHidden && this.setState({ isHidden });
```
Might be also worthwhile to look into throttling resize events (only call function after 500ms or something like that

Reply:
Thanks for the suggestion - I've updated as suggested above and implemented debounce utility function on resize.